shardingsphere-sql-parser/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4

```diff
@@ -303,7 +303,14 @@ dataType
     ;
 
 dataTypeName
-    : identifier
+    : INT | SMALLINT | INTEGER | BIGINT | DECIMAL | NUMERIC | REAL | FLOAT | doublePrecision | SMALLSERIAL | SERIAL | BIGSERIAL
+    | MONEY | VARCHAR | CHARACTER | CHAR | TEXT | NAME | BYTEA | TIMESTAMP | DATE | TIME | INTERVAL | BOOLEAN | ENUM | POINT
+    | LINE | LSEG | BOX | PATH | POLYGON | CIRCLE | CIDR | INET | MACADDR | BIT | VARBIT | TSVECTOR | TSQUERY | UUID | XML
+    | JSON | INTRANGE | NUMRANGE | TSRANGE | TSTZRANGE | DATERANGE | ARRAY | identifier
+    ;
+
+doublePrecision
```

Comment:
Why does `doublePrecision` not exist in `PostgreSQLKeyword.g4 `?

Reply:
> Why does `doublePrecision` not exist in `PostgreSQLKeyword.g4 `?

DOUBLE and PRECISION exist in KeyWord.g4, so I added an expression to represent double precision. Now I find that it is better to add keywords directly, I will change to keyword and remove doublePrecision.
`DOUBLEPRECISION
    : D O U B L E [ ]+ P R E C I S I O N
    ;`